classes/item.rb

```diff
@@ -0,0 +1,40 @@
+# # Item class represents a generic catalog item.
+
+class Item
+  attr_accessor :title, :publish_date
+  attr_reader :id, :archived
```

Comment:
- [ ] Kindly ensure that your `Item Class` includes `attr_reader` for these attributes. :wink: 

```ruby
  attr_reader :genre
  attr_reader :author
  attr_reader :label
```